src/main/java/org/traccar/WebDataHandler.java

```diff
@@ -59,6 +59,7 @@
 
     private static final String KEY_POSITION = "position";
     private static final String KEY_DEVICE = "device";
+    private static final String ATTRIBUTE_DEVICE_ALLOW_FORWARDING = "deviceAllowForwarding";
```

Comment:
If it's a device attribute, you don't need to add a prefix "device".

Reply:
I did the same as some other attributes for the device

```bash
    data: [{
        key: 'web.reportColor',
        name: Strings.attributeWebReportColor,
        valueType: 'color'
    }, {
        key: 'devicePassword',
        name: Strings.attributeDevicePassword,
        valueType: 'string'
    }, {
        key: 'processing.copyAttributes',
        name: Strings.attributeProcessingCopyAttributes,
        valueType: 'string'
    }, {
        key: 'decoder.timezone',
        name: Strings.sharedTimezone,
        valueType: 'string',
        dataType: 'timezone'
    }, {
        key: 'deviceInactivityStart',
        name: Strings.attributeDeviceInactivityStart,
        valueType: 'number',
        allowDecimals: false,
        minValue: 1
    }, {
        key: 'deviceInactivityPeriod',
        name: Strings.attributeDeviceInactivityPeriod,
        valueType: 'number',
        allowDecimals: false,
        minValue: 1
    }, {
        key: 'deviceAllowForwarding',
        name: Strings.attributeDeviceAllowForwarding,
        valueType: 'boolean'
    }]
```
I can remove the prefix. Let me know which you want?

Reply:
We only use prefix when it's not very obvious what attribute does without prefix. With forwarding it seems pretty straightforward, so I think we should remove prefix.

Reply:
Removed the prefix.